.github/actions/setup-npm-demos-npm/action.yml

```diff
@@ -0,0 +1,24 @@
+name: "Setup demo npm"
```

Comment:
Food for thought: we use this here and in optable sandbox maybe this can become a shared actions at some point.

Reply:
We could make a reusable version but we would need to pass a bunch of parameters. I did consider it and decided to copy for now but I agree it could become a versioned reusable shared action.